source/components/index.markdown

```diff
@@ -88,8 +88,19 @@ Entities are things that you want to observe within Home Assistant. Support for
   <td>Track what is being played and control playback. (temporarely disabled awaiting protobuf 3 release)</td>
 </tr>
 
+<tr>
```

Comment:
It is the same right now with the Vera and Wink hubs.

The question is if people go through this list to:
 * see how to support certain technologies
 * see if their existing hardware is supported

Anyway, open for suggestions.